docs/charts/radar.md

```diff
@@ -152,12 +152,17 @@ The radar chart defines the following configuration options. These options are m
 ## Scale Options
 
 The radar chart supports only a single scale. The options for this scale are defined in the `scale` property.
+The options for `scale` property can be refererenced from the [Linear Radial Axis page](../axes/radial/linear).
 
 ```javascript
 options = {
     scale: {
         // Hides the scale
-        display: false
+        display: false,
+        ticks: {
```

Comment:
This is a little bit of a weird example because there would be no reason to change the `suggestedMin` and `suggestedMax` if you're not showing the scale. Can we use a more realistic example?

Maybe change:
```
// Hides the scale
display: false,
```

To:
```
angleLines: {
  display: false
},
```